src/main/kotlin/lotto/model/number/LottoNumbers.kt

```diff
@@ -0,0 +1,27 @@
+package lotto.model.number
+
+import java.util.TreeSet
+
+data class LottoNumbers private constructor(private val lottoNumbers: TreeSet<LottoNumber>) : Set<LottoNumber> by lottoNumbers {
+    init {
+        require(lottoNumbers.size == CANDIDATE_SIZE) { "로또 티켓은 6개의 로또 숫자로 이루어져 있다" }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        require(lottoNumbers.size == CANDIDATE_SIZE) { "로또 티켓은 ${CANDIDATE_SIZE}개의 로또 숫자로 이루어져 있다" }
```